blocitoff/app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Skeleton, auto-generated files like this (`users.coffee`, `users.scss`) can safely be removed. 

Get in the habit of reviewing all files generated by the `rails g(enerate) ...` command and delete those that aren't needed (they can always be generated again later if it turns out they're needed). 

*Before* `git add`-ing, use `git status` to review all files that have changed or have been created.

In particular, open every newly-generated file and try to determine if its needed.

If a newly-generated file is not needed, then do not `git add` it. Instead delete it (you can use `rm path/to/unwanted/file.extension` to delete a file from the command line).

By keeping the app free of unnecessary files like this it tends to make maintenance and debugging a little easier as the app contains less clutter.